meshtastic/tests/test_ble_compat_services_coverage.py

```
@@ -0,0 +1,1273 @@
+
+        # The actual handler behavior is tested in management_runtime tests
+        # Just verify the shim structure is correct by checking the call doesn't crash
+        # on initial handler resolution. Due to complex mock interactions, we accept
+        # either success or a controlled failure.
+        try:
+            result = BLEManagementCommandsService._acquire_client_for_target(
+                iface,
+                address=None,
+                target_address="AA:BB:CC:DD:EE:FF",
+                expected_implicit_binding=None,
+                connected_elsewhere=lambda key, owner=None: False,
```

Ruff (0.15.7)

[warning] Unused lambda argument: `key`

(ARG005)